src/Stratis.BreezeD/Program.cs

```diff
@@ -29,7 +31,7 @@ public class Program
 
                 if (isStratis)
                 {
-                    Network network = isTestNet ? Networks.StratisTest : Networks.StratisMain;
+                    Network network = isTestNet ? NetworkRegistration.Register(new StratisTest()) : NetworkRegistration.Register(new StratisMain());
```

Comment:
I wonder if we should actually register a network when creating it, so inside the constructor of StratisTest() for example, syncing networks now.